Preliminary results for the Dingemans case

ezve opened the pull request:
The comparison of the free-surface elevation between the results of Proteus and the experimental data for 3 different gauge locations are shown in the following figures. The refinement is set to λ/65 which is relatively coarse. 
     ![1](https://cloud.githubusercontent.com/assets/8614856/8105358/8b0f27a8-1030-11e5-98b2-   d39205a8caad.JPG)
![2](https://cloud.githubusercontent.com/assets/8614856/8105360/8bf81594-1030-11e5-966f-dc6c6430578e.JPG)
       ![3](https://cloud.githubusercontent.com/assets/8614856/8105361/8cc4afe6-1030-11e5-809c-4a8d6f5cc70b.JPG)

adimako commented:
@cekees I think this is artificial breaking due to lack of resolution. A mesh with lambda/Dx ~ 200 would be more appropriate for this
@ezve Does Dingemans benchmark have some cases with longer waves? It might be easier for them to model

cekees commented:
Should I kick off a simulation on one of the clusters with more resolution?

ezve commented:
@adimako yes there are two more cases I will upload them.

adimako commented:
Yeah I think it is worth because currenty we cannot tell if there are other problems, as the resolution is too coarse

ezve commented:
@cekees @adimako I have added the rest of the wave conditions tested in the experiments (noted as _02 and _03)

cekees commented:
Thanks! I'll try to get them going this afternoon.

On Thu, Jun 11, 2015 at 1:37 PM, ezve <notifications@github.com> wrote:

> @cekees <https://github.com/cekees> @adimako <https://github.com/adimako>
> I have added the rest of the wave conditions tested in the experiments
> (noted as _02 and _03)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/erdc-cm/air-water-vv/pull/96#issuecomment-111117645>.
>

ezve commented:
@cekees @adimako Here are the results for the first case (_01) and for two refinement levels comparing to the experimental results
![legend](https://cloud.githubusercontent.com/assets/8614856/8554967/108b7ff8-24e5-11e5-92c2-cabee32c8534.JPG)
![dig_1a](https://cloud.githubusercontent.com/assets/8614856/8554970/1453366c-24e5-11e5-9384-8fdea29a1b71.JPG)
![dig_1b](https://cloud.githubusercontent.com/assets/8614856/8554971/15a1cbb4-24e5-11e5-9e41-1205dcdcfa79.JPG)
![dig_1c](https://cloud.githubusercontent.com/assets/8614856/8554974/17f78da4-24e5-11e5-9419-dcc622e3a327.JPG)

ezve commented:
@cekees @adimako Here are the results for the third case (_03) and for a single refinement level
![legend](https://cloud.githubusercontent.com/assets/8614856/8555048/965cd4b0-24e5-11e5-9624-d0f1ce1e8ab1.JPG)
![dig_3a](https://cloud.githubusercontent.com/assets/8614856/8555051/99bf5272-24e5-11e5-9af0-d0f71417728b.JPG)
![dig_3b](https://cloud.githubusercontent.com/assets/8614856/8555053/9d195ab2-24e5-11e5-9230-e5a87292fa2c.JPG)
![dig_3c](https://cloud.githubusercontent.com/assets/8614856/8555055/9f607bc0-24e5-11e5-93c1-3b2b51ca11c0.JPG)

@cekees I didn't find the vof integration probe output for the second case (_02) in the files you have sent us. Can you please check if you have it?

ezve commented:
@cekees @adimako Here are the results for the second case (_02) and for a single refinement level

![legend](https://cloud.githubusercontent.com/assets/8614856/8571939/8679dc5e-2581-11e5-9028-07adfed89e8a.JPG)
![dig_1b](https://cloud.githubusercontent.com/assets/8614856/8571941/869bb8d8-2581-11e5-8834-62d52be42c1d.JPG)
![dig_2b](https://cloud.githubusercontent.com/assets/8614856/8571942/86a2a3b4-2581-11e5-8dfb-4b29703958c4.JPG)
![dig_3b](https://cloud.githubusercontent.com/assets/8614856/8571938/866d7bc6-2581-11e5-820b-8fa41ab0ea5a.JPG)

adimako commented:
@ezve, can you post the proof that the velocity in the generation is pointing the opposite direction we would expect?

ezve commented:
@adimako I have created the following pull request concerning this https://github.com/erdc-cm/air-water-vv/pull/95

ezve commented:
@adimako  The pull request mentioned above illustrates the problem for the linear wave case, in the following figures you can see the same behaviour for the non-linear wave case. The first figure shows the horizontal velocity at x=0 (left boundary) and y=-0.5 relatively to the mean water level; and the second figure shows the corresponding vertical velocity.
 
![nonlinearwaves](https://cloud.githubusercontent.com/assets/8614856/8829595/63461e50-308f-11e5-8cc3-0c1cf3748004.JPG)

![nonlinearwaves_v](https://cloud.githubusercontent.com/assets/8614856/8829571/3e16855c-308f-11e5-8349-d7d7ef7f8326.JPG)

cekees commented:
Also, @ezve , on this issue with the v-component of velocity. Does it show up the same if when using gauges and extracting results from ParaView?

cekees commented:
Aggelos, this is Lara, Chris's wife. We're on the road to Oxford from Wales. We were wondering if the party were still happening. If it is, we will likely be late because of traffic. Feel free to call or text a reply. Those are more reliable than email right now, I think. Thanks!                                                                                                                                                                                                                                                                                                                                        Sent from my BlackBerry 10 smartphone on the Verizon Wireless 4G LTE network.

cekees commented:
@adimako , since the velocity issue turned out to be a bug in the plotting script unrelated to the input files I think the results I've run on finer grids should be up to date. I will try to post-process today. I recall that the next thing you suggested was running at a smaller time step since the celerity is faster than the velocity. @ezve mentioned that she was already doing some phase matching of the above results so I'm not quite clear on how much real dispersion error there is in these.

adimako commented:
@cekees What I am afraid is that dispersion error is shortening the wave lenght and causes premature wave breaking. If @ezve can give us an estimate of the accumulative phase lag of the peaks we can figure out more or less how much it will be. The other thing we may want to consider is switching off turbulence models, at least for the non breaking cases (case 01 and 03)

ezve commented:
@adimako @cekees I have updated my comment above with regard to the results of the first case (_01). I have also added the initial part of the comparisons between the numerical and experimental results.

cekees commented:
The runs @ezve and I submit all seem to hang during the setup of the gauges.
